docs/en/ingest-management/elastic-agent/install-elastic-agent-in-container.asciidoc

```diff
@@ -12,6 +12,8 @@ To learn how to run {agent}s in a containerized environment, see:
 
 * <<running-on-kubernetes-managed-by-fleet>>
 
+** <<running-on-eks-managed-by-fleet>>
```

Comment:
link to the view: https://observability-docs_2360.docs-preview.app.elstc.co/guide/en/fleet/master/install-elastic-agents-in-containers.html